### PR TITLE
Added an option to edit the preview confirm and cancel text

### PR DESCRIPTION
**Pull request by ElBananaa**

### CODE REVIEW

File: src/main/java/com/bgsoftware/superiorskyblock/island/SIslandPreview.java [hunk 1]
```diff
@@ -24,11 +24,11 @@ public SIslandPreview(SuperiorPlayer superiorPlayer, Location previewLocation, S
         this.islandName = islandName;
 
         PlayerChat.listen(superiorPlayer.asPlayer(), message -> {
-            if(message.equalsIgnoreCase("CONFIRM")){
+            if(message.equalsIgnoreCase(Locale.ISLAND_PREVIEW_CONFIRM_TEXT.toString())){
```
Comment (OmerBenGera) [inline src/main/java/com/bgsoftware/superiorskyblock/island/SIslandPreview.java]: This should be changed to:
`if(message.equalsIgnoreCase(Locale.ISLAND_PREVIEW_CONFIRM_TEXT.getMessage(<user-locale>)))`

Comment (ElBananaa) [reply src/main/java/com/bgsoftware/superiorskyblock/island/SIslandPreview.java]: I completely forgot that users can edit their lang. It should be ok now.

**Review by OmerBenGera — APPROVED** [review]

**Comment by OmerBenGera** [comment]
I forgot to ask for one more thing - please add to all the lang files the English version of the new messages, so they don't miss the messages.

**Comment by ElBananaa** [comment]
> I forgot to ask for one more thing - please add to all the lang files the English version of the new messages, so they don't miss the messages.

Alright, done. :)

**Pull request merged**